### DISCUSSION TITLE
vr mirroring does not work anymore in firefox 79

### DISCUSSION THREAD

**Discussion by massimo-bortolamei-hegias** [mozilla/hubs#2808] [General Questions & Discussion] [answered]
Have you noticed that since Firefox 79 rolled out, the mirroring of VR into the browser does not work anymore? 
In hubs I get black screen, in simple AFrame examples (like this one https://aframe.io/aframe/examples/boilerplate/hello-world/) I get white screen. What's going on?

**Comment by brianpeiris** [comment, marked as answer]
Hi @massimo-bortolamei-hegias. If you're running into this with other WebVR apps, you might want to [file a bug report against Firefox itself](https://bugzilla.mozilla.org/enter_bug.cgi?product=Core&component=WebVR&status_whiteboard=[webvr]&cc=kgilbert@mozilla.com). 
I'd suggest also testing with the simplest WebVR sample to confirm it's an issue with WebVR and not Hubs specifically: https://webvr.info/samples/04-simple-mirroring.html

**Reply by emclaren** [reply]
Hi @massimo-bortolamei-hegias I'm going to mark this as the answer for now, but if you find you can open this https://webvr.info/samples/04-simple-mirroring.html but can't use Hubs, please let us know so we can continue troubleshooting

**Comment by massimo-bortolamei-hegias** [comment]
Hi guys, thank you for your answer and sorry for the delay. I see the problem with the webvr example you linked as well. 
I'll go ahead and file a bug report against Firefox.

**Reply by massimo-bortolamei-hegias** [reply]
https://bugzilla.mozilla.org/show_bug.cgi?id=1662168
